pkgs/applications/video/kodi/default.nix

```diff
@@ -194,6 +194,13 @@ in stdenv.mkDerivation {
       autoconf automake libtool # still needed for some components. Check if that is the case with 19.0
     ] ++ lib.optionals useWayland [ wayland-protocols ];
 
+    patches = [
+      (fetchpatch {
```

Comment:
Can you link a reason, or comment a reason why we need this patch?
And when it can likely be removed, like `remove in > *.x.*`?

Reply:
Probably something like the following (which is what I had before checking if somebody was already working on this... oops):

```
      # Adds missing cassert includes, fixing builds. This will be unnecessary
      # after 18.6 is released (which will contain this patch)
```

Reply:
Used your comment, thanks, @cole-h :).